lang/po/new/cs.po

```diff
@@ -1663,7 +1659,7 @@ msgstr "Probiha zmena modu..."
 # MSG_MODEL c=8
 #: messages.c:132
 msgid "Model"
-msgstr ""
+msgstr "Model"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.